prometheus_example.yml

```diff
@@ -24,4 +24,4 @@ scrape_configs:
 
   - job_name:       'ss-server'
     static_configs:
-      - targets: ['localhost:8080']
+      - targets: ['localhost:9091']
```

Comment:
This is unrelated, I think it should be changed to match with the tutorial in the README

Reply:
👍